libproxy/libproxy.map

```diff
@@ -1,9 +1,9 @@
-LIBPROXY {
+LIBPROXY_0.4.16 {
```

Comment:
Nope..

You must get a new block, not change the original block - the original block remains a LIBPROXY (unversioned, to show that's the starting point)

You can never add a new API into an existing block

Reply:
Nope again (unless I massively misunderstand how it works)

No released version of libproxy ever had symbol versioning, so what is done here does not matter yet.
All symbols in libproxy are exported as unversioned symbols anyway (the versioned symbols are just like aliases):
```
   134: 00000000000052c0    66 FUNC    GLOBAL DEFAULT   14 px_proxy_factory_free_proxies@@LIBPROXY_0.4.16
   405: 00000000000052c0    66 FUNC    GLOBAL DEFAULT   14 px_proxy_factory_free_proxies
```

Reply:
it does matter to me

Reply:
Adding a new block just for a new function without any relation to a version doesn't make much sense.

Reply:
it does to me

The version table can then serve as easy lookup again; the existing three api calls were there 'from the beginning' (literally) - hence in the block 'LIBPROXY'; the new API appeared in LIBPROXI_0_4_16

Reply:
I agree with that, but it does not make much sense if looking at the linker perspective - the versioned symbols are only there from libproxy 0.4.16 on. So we could say that those three symbols have version "LIBPROXY_0.0.1", but that would result in these errors if a too old version of libproxy is used:

```Symbol "px_proxy_factory_new@LIBPROXY_0.0.1" not found in "/usr/lib64/libproxy.so.0.4.15"```

which is highly misleading as 0.4.15 > 0.0.1.

The symbol version is from that point on part of the API, so the versioning itself is part of the versioning.